heron/api/src/java/shade.conf

```diff
@@ -0,0 +1 @@
+rule com.google** com.twitter.heron.shaded.@0
```

Comment:
@nlu90 what do you think? Do you remember the "com.twitter.heron.shaded.do.not.use" that I showed you during the stand-up yesterday?